src/pupil_labs/simple_api_check.py

```diff
@@ -0,0 +1,64 @@
+from pupil_labs.realtime_api.simple import discover_devices
+from pupil_invisible_lsl_relay import pi_gaze_relay
```

Comment:
The import needs to be changed accordingly to import from this repository
```suggestion
from pupil_labs.invisible_lsl_relay import pi_gaze_relay
```